int8 number_robot
int8 level
int8 level_elevator
bool free_elevator
bool robot_finish

